dolphinscheduler-service/src/main/java/org/apache/dolphinscheduler/service/cron/CronUtils.java

```diff
@@ -147,7 +143,7 @@ public static List<ZonedDateTime> getFireDateList(@NonNull ZonedDateTime startTi
         List<ZonedDateTime> dateList = new ArrayList<>();
         ExecutionTime executionTime = ExecutionTime.forCron(cron);
 
-        while (Stopper.isRunning()) {
+        while (!ServerLifeCycleManager.isStopped()) {
```

Comment:
Should we check server status here? Maybe it's no need.

Reply:
Yes, we don't need to check the server status in all places.